dandiapi/api/migrations/0031_embargoedzarrarchive_checksum_zarrarchive_checksum_and_more.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 4.0.6 on 2022-07-07 21:36
+
+from django.db import migrations, models
+
+
+def forward_set_checksum(apps, schema_editor):
+    ZarrArchive = apps.get_model('api', 'ZarrArchive')  # noqa: N806
+    zarrs = ZarrArchive.objects.filter(checksum__isnull=True)
+    for zarr in zarrs.iterator():
+        zarr.checksum = None
+        zarr.status = 'Pending'
+        zarr.save(update_fields=['checksum', 'status'])
```

Comment:
Can't this be done in a single query with

```suggestion
    zarrs.update(checksum=None, status='Pending')
```

Reply:
Yep! Good catch, thanks.